fgrpc/grpcrunner.go

```diff
@@ -251,34 +251,49 @@ func grpcDestination(dest string) (parsedDest string) {
 	// and set the port number.
 	switch {
 	case strings.HasPrefix(dest, prefixHTTP):
-		parsedDest = strings.Replace(dest, prefixHTTP, "", 1)
+		parsedDest = strings.TrimSuffix(strings.Replace(dest, prefixHTTP, "", 1), "/")
 		port = defaultHTTPPort
 		log.Infof("stripping http scheme. grpc destination: %v: grpc port: %s",
 			parsedDest, port)
 	case strings.HasPrefix(dest, prefixHTTPS):
-		parsedDest = strings.Replace(dest, prefixHTTPS, "", 1)
+		parsedDest = strings.TrimSuffix(strings.Replace(dest, prefixHTTPS, "", 1), "/")
 		port = defaultHTTPSPort
 		log.Infof("stripping https scheme. grpc destination: %v. grpc port: %s",
 			parsedDest, port)
 	default:
 		parsedDest = dest
 		port = DefaultGRPCPort
 	}
-	if _, _, err := net.SplitHostPort(parsedDest); err == nil {
+	host, sPort, err := net.SplitHostPort(parsedDest)
+	// parsedDest is in the form of host:port or ip:port
+	if err == nil {
```

Comment:
why is all this coming back/changing - should just be a simple change with just the suffix trim above ?